packages/app/src/components/App.js

```diff
@@ -123,8 +123,10 @@ export class App extends Component {
                     <div className="item6 interpretations">
                         Interpretations panel
                     </div>
-                    <div className="item8 canvas">
+                    <div className="title-bar">
                         <TitleBar />
+                    </div>
+                    <div className="item8 canvas">
```

Comment:
you probably don't want `item8`, right?

Reply:
no, but those are already removed in the other branch